### PR TITLE
feat(components-native): Add size variation for progress bar

### PR DESCRIPTION
**Pull request by jjwlee94**
## Motivations
- To allow for a smaller size as we do with the progress bar for web.

### Added
- Size prop on `ProgressBar`.

| Small | Base (default) |
| ----- | ----- |
| image | image |

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -34,7 +35,12 @@ export function ProgressBar({
           inProgress={inProgress}
         />
       ) : (
-        <View style={styles.progressBarContainer}>
+        <View
+          style={[
+            styles.progressBarContainer,
+            size === "small" && { height: 10 },
```
Comment (chris-at-jobber) [inline packages/components-native/src/ProgressBar/ProgressBar.tsx:41]: We should reference the same sizes as web:
image

```suggestion
            size === "small" && { height: 8 },
```

**Review by chris-at-jobber — COMMENTED** [review]
Design approved ✅ 

@Aiden-Brine mind taking a look for code review?

**Comment by Aiden-Brine** [comment]
Hey @jjwlee94, thank you very much for making contributions to Atlantis and I really like the idea of allowing for the size options on mobile to match those on web. I pushed up a new commit that modifies your changes a bit but should give you the same outcome you want (a progress bar with a height of 8px). Here is an explanation of my changes and please let me know what you think 😄 
- Currently web has three size options smaller (4px), small (8px), and base (16px)
- Before your changes mobile was set to exclusively 20px so not consistent with web at all
- The good news is, ProgressBar is not currently being used in jobber-mobile so we can make breaking changes to it 🎉 
- So my changes give size options of smaller (4px), small (8px), and base (16px) to the mobile version in a way that is more consistent with our other Atlantis components

Please let me know if you have any questions. Also I would appreciate another design review from you @chris-at-jobber 🙏

File: packages/components-native/src/ProgressBar/ProgressBar.size.style.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { StyleSheet } from "react-native";
+import { tokens } from "../utils/design";
+
+export const sizeStyles = StyleSheet.create({
+  smaller: { height: tokens["space-smaller"] },
```
Comment (chris-at-jobber) [inline packages/components-native/src/ProgressBar/ProgressBar.size.style.ts:5]: `small` 😉 change: no longer using [spacing for size values!](https://atlantis.getjobber.com/?path=/docs/guides-frontend-styleguide--docs#size-width-and-height)

**Review by chris-at-jobber — COMMENTED** [review]
Design approved _(again, one small nit on the size values code but we could even follow up on this in a separate PR where we do the same in web 📏 )_

**Review by Aiden-Brine — APPROVED** [review]

**Pull request merged**